mrp_bom_attribute_match/reports/mrp_report_bom_structure.py

```diff
@@ -31,9 +31,7 @@ def _get_bom_lines(self, bom, bom_quantity, product, line_id, level):
             if to_ignore_line_ids:
                 for to_ignore_line_id in to_ignore_line_ids:
                     bom.bom_line_ids = [(3, to_ignore_line_id, 0)]
-        product = bom.get_component_template_product(
-            line, product, line.product_id
-        )
+        product = bom.get_component_template_product(line, product, line.product_id)
```

Comment:
Please check the function name, shouldn't it start with the underscore?

Maybe there is something I am missing here